### ISSUE TITLE
Configurable LACP policy 

### ISSUE THREAD

**Issue by xen0l** [joyent/smartos-live#530]
SmartOS should provide a directive within config file for setting LACP policy, e.g. aggr0_lacp_policy.

**Comment by amalleo25** [comment]
Agreed, I would like to see this as well. One workaround is to create a transient service that will modify the aggr policy on startup but it would be nice have this officially supported in the config file.

**Comment by bahamat** [comment]
Related to [NET-374](https://smartos.org/bugview/NET-374).
